oneview_redfish_toolkit/api/computer_system.py

```diff
@@ -300,6 +300,13 @@ def build_server_profile(profile_name,
                     server_profile_template,
                     external_storage_block)
 
+        key = 'complianceControl'
+        for item in server_profile.keys():
+            if isinstance(server_profile[item], dict):
+                if (key in server_profile[item].keys()):
+                    server_profile[item].pop(key)
+
+
```

Comment:
Can you group this code together where we pop other attributes. Check the line no. 279.

Reply:
Done, Clubbed all the pop statement towards the end of the method.

Performed compose operation after the change, Test successful
![image](https://user-images.githubusercontent.com/55734315/96888917-bb53fb00-14a3-11eb-9eb0-aa392d12638d.png)